crates/provider-wit-bindgen/src/lib.rs

```diff
@@ -504,29 +1317,42 @@ impl WitBindgenOutputVisitor {
 
 impl VisitMut for WitBindgenOutputVisitor {
     fn visit_item_mod_mut(&mut self, node: &mut ItemMod) {
-        // Save the WIT namespace that we've recognized
+        #[cfg(feature = "debug")]
+        eprintln!(
+            "[info] (bindgen module hierarchy): {}> {}",
```

Comment:
Could you help me understand why we cannot use the standard Rust logging functionality like the `tracing` crate here?

Reply:
See previous comment(s) and Connor's post, closing this comment